packages/ember-htmlbars/tests/helpers/unbound_test.js

```diff
@@ -1,595 +0,0 @@
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile('{{unbound (-surround prefix value "bar")}} {{-surround prefix value "bar"}} {{unbound (-surround "bar" value suffix)}} {{-surround "bar" value suffix}}'),
-    context: EmberObject.create({
-      prefix: 'before',
-      value: 'core',
-      suffix: 'after'
-    })
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'before-core-bar before-core-bar bar-core-after bar-core-after', 'first render is correct');
-  run(function() {
-    set(view, 'context.prefix', 'beforeChanged');
-    set(view, 'context.value', 'coreChanged');
-    set(view, 'context.suffix', 'afterChanged');
-  });
-  equal(view.$().text(), 'before-core-bar beforeChanged-coreChanged-bar bar-core-after bar-coreChanged-afterChanged', 'only bound values change');
-});
-
-QUnit.test('should be able to render unbound forms of multi-arg helpers', function() {
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile('{{-fauxconcat foo bar bing}} {{unbound (-fauxconcat foo bar bing)}}'),
-    context: EmberObject.create({
-      foo: 'a',
-      bar: 'b',
-      bing: 'c'
-    })
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'abc abc', 'first render is correct');
-
-  run(function() {
-    set(view, 'context.bar', 'X');
-  });
-
-  equal(view.$().text(), 'aXc abc', 'unbound helpers/properties stayed the same');
-});
-
-QUnit.test('should be able to render an unbound helper invocation for helpers with dependent keys', function() {
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile('{{-capitalizeName person}} {{unbound (-capitalizeName person)}} {{-concatNames person}} {{unbound (-concatNames person)}}'),
-    context: EmberObject.create({
-      person: EmberObject.create({
-        firstName: 'shooby',
-        lastName:  'taylor'
-      })
-    })
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'SHOOBY SHOOBY shoobytaylor shoobytaylor', 'first render is correct');
-
-  run(function() {
-    set(view, 'context.person.firstName', 'sally');
-  });
-
-  equal(view.$().text(), 'SALLY SHOOBY sallytaylor shoobytaylor', 'only bound values change');
-});
-
-QUnit.test('should be able to render an unbound helper invocation in #each helper', function() {
-  view = EmberView.create({
-    [OWNER]: owner,
-    template: compile(
-      ['{{#each people as |person|}}',
-       '{{-capitalize person.firstName}} {{unbound (-capitalize person.firstName)}}',
-       '{{/each}}'].join('')),
-    context: {
-      people: emberA([
-        {
-          firstName: 'shooby',
-          lastName:  'taylor'
-        },
-        {
-          firstName: 'cindy',
-          lastName:  'taylor'
-        }
-      ])
-    }
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'SHOOBY SHOOBYCINDY CINDY', 'unbound rendered correctly');
-});
-
-QUnit.test('should be able to render an unbound helper invocation with bound hash options', function() {
```

Comment:
:heavy_check_mark: should be able to render an unbound helper invocation with bound hash options